src/services/API/User.js

```diff
@@ -42,15 +47,14 @@ export default {
   },
 
   getUserConfig(userId) {
-    return apiClient().get(
-      process.env.VUE_APP_BACKEND_USER_CONFIG + "?user-id=" + userId
-    );
+    // retrieves the config for the given user ID
+    return apiClient().get(usersEndpoint + userId + userConfigEndpoint);
```

Comment:
What vaibhav did for plios, that the user will already be available and we dont need to pass it, can't that be done here as well?